rules/windows/defense_evasion_clearing_windows_security_logs.toml

```diff
@@ -56,7 +56,8 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.action:("audit-log-cleared" or "Log clear") and winlog.api:"wineventlog"
+event.action:("audit-log-cleared" or "Log clear") and winlog.api:"wineventlog" and
+  not winlog.provider_name:"AD FS Auditing"
```

Comment:
https://github.com/elastic/beats/issues/27288 seems to be solved, but I still see a lot of hits in telemetry so I'm excluding it here